packages/slo-stat-bot/test/slo-stat-label.ts

```diff
@@ -0,0 +1,305 @@
+import {Probot} from 'probot';
+import nock from 'nock';
+import * as fs from 'fs';
+import {expect} from 'chai';
+import {describe, it, beforeEach} from 'mocha';
+import Webhooks from '@octokit/webhooks';
+import snapshot from 'snap-shot-it';
+
+import handler from '../src/slo-stat-label';
+import spies from 'chai-spies';
+import { doesNotReject } from 'assert';
+
+const chai = require('chai');
+chai.use(spies);
+const sinon = require('sinon');
+
+
+nock.disableNetConnect();
+
+const fixturesPath = resolve(__dirname, '../../test/fixtures');
+
+describe('slo-status-label', () => {
+  let probot: Probot;
+
+  beforeEach(() => {
+    probot = new Probot({
+      // use a bare instance of octokit, the default version
+      // enables retries which makes testing difficult.
+      // eslint-disable-next-line node/no-extraneous-require
+      Octokit: require('@octokit/rest'),
+    });
+
+    probot.app = {
+      getSignedJsonWebToken() {
+        return 'abc123';
+      },
+      getInstallationAccessToken(): Promise<string> {
+        return Promise.resolve('abc123');
+      },
+    };
+    probot.load(handler);
+  });
+
+  describe('opened or reopened pull request', () => {
+    let payload: Webhooks.WebhookPayloadPullRequest;
+    let stub: any;
+    
+    beforeEach(() => {
+      payload = require(resolve(
+        fixturesPath,
+        'events',
+        'pull_request_opened'
+      ));
+      stub = sinon.stub(handler, 'handle_slos');
+    });
+
+    afterEach(function () {
+      sinon.restore();
+      nock.cleanAll;
+    });
+
+    it('Error is logged if get list of files fails', async () => {
+      const requests = nock('https://api.github.com')
+        .get('/repos/testOwner/testRepo/pulls/6/files?per_page=100')
+        .reply(404)
+
+      await probot.receive({name: 'pull_request.opened', payload, id: 'abc123',});
+      requests.done();
+    });
+
+    it('triggers handle_slos function since issue_slo_rules.json is present', async () => {
+      const containsSloFile = require(resolve(
+        fixturesPath,
+        'events',
+        'contains_slo_file'
+      ));
+      const requests = nock('https://api.github.com')
+        .get('/repos/testOwner/testRepo/pulls/6/files?per_page=100')
+        .reply(200,containsSloFile)
+      await probot.receive({name: 'pull_request.opened', payload, id: 'abc123',});
+
+      expect(stub.called).to.equals(true);
+      requests.done();
+    });
+
+    it('does not trigger handle_slos function since issue_slo_rules.json is not present', async () => {
+      const doesNotContainSLOFile = require(resolve(
+        fixturesPath,
+        'events',
+        'not_contains_slo_file'
+      ));
+      const requests = nock('https://api.github.com')
+        .get('/repos/testOwner/testRepo/pulls/6/files?per_page=100')
+        .reply(200,doesNotContainSLOFile)
+
+      await probot.receive({name: 'pull_request.opened', payload, id: 'abc123',});
+
+      expect(stub.called).to.equals(false);
```

Comment:
As opposed to using expect here, use sinon.assert.calledOnce(stub)

Reply:
This test is looking to see if "issue_slo_rules.json" is not present, then the handles.slo rules is not called. So I can use sinon.assert.notCalled(stub) instead.